extensions/mentions/extend.php

```diff
@@ -126,23 +124,5 @@
 
     // Tag mentions
     (new Extend\Conditional())
-        ->whenExtensionEnabled('flarum-tags', [
-            (new Extend\Formatter)
-                ->render(Formatter\FormatTagMentions::class)
-                ->unparse(Formatter\UnparseTagMentions::class),
-
-            (new Extend\ApiSerializer(BasicPostSerializer::class))
-                ->hasMany('mentionsTags', TagSerializer::class),
-
-            (new Extend\ApiController(Controller\ShowDiscussionController::class))
-                ->load(['posts.mentionsTags']),
-
-            (new Extend\ApiController(Controller\ListDiscussionsController::class))
-                ->load([
-                    'firstPost.mentionsTags', 'lastPost.mentionsTags',
-                ]),
-
-            (new Extend\ApiController(Controller\ListPostsController::class))
-                ->load(['mentionsTags']),
-        ]),
+        ->whenExtensionEnabled('flarum-tags', TagExtender::class),
```

Comment:
please keep this inside the `extend.php` file as it's easier to read. with a simple `fn () => [...]`